src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -76,96 +82,103 @@
   public static final String TRIPLEA_SERVER_OBSERVER_JOIN_WAIT_TIME = "triplea.server.observerJoinWaitTime";
   // non-commandline-argument-properties (for preferences)
   // first time we've run this version of triplea?
-  private static final String TRIPLEA_FIRST_TIME_THIS_VERSION_PROPERTY =
-      "triplea.firstTimeThisVersion" + ClientContext.engineVersion();
-  private static final String TRIPLEA_LAST_CHECK_FOR_ENGINE_UPDATE = "triplea.lastCheckForEngineUpdate";
-  // only for Online?
-  public static final String TRIPLEA_MEMORY_ONLINE_ONLY = "triplea.memory.onlineOnly";
-  // what should our xmx be approximately?
-  public static final String TRIPLEA_MEMORY_XMX = "triplea.memory.Xmx";
-  public static final String TRIPLEA_MEMORY_USE_DEFAULT = "triplea.memory.useDefault";
-  public static final String SYSTEM_INI = "system.ini";
+  private static final String SYSTEM_INI = "system.ini";
   public static final int MINIMUM_CLIENT_GAMEDATA_LOAD_GRACE_TIME = 20;
-  public static final int DEFAULT_CLIENT_GAMEDATA_LOAD_GRACE_TIME =
+  private static final int DEFAULT_CLIENT_GAMEDATA_LOAD_GRACE_TIME =
       Math.max(MINIMUM_CLIENT_GAMEDATA_LOAD_GRACE_TIME, 25);
   // need time for network transmission of a large game data
   public static final int MINIMUM_SERVER_OBSERVER_JOIN_WAIT_TIME = MINIMUM_CLIENT_GAMEDATA_LOAD_GRACE_TIME + 10;
-  public static final int DEFAULT_SERVER_OBSERVER_JOIN_WAIT_TIME =
+  private static final int DEFAULT_SERVER_OBSERVER_JOIN_WAIT_TIME =
       Math.max(DEFAULT_CLIENT_GAMEDATA_LOAD_GRACE_TIME + 10, 35);
   public static final int ADDITIONAL_SERVER_ERROR_DISCONNECTION_WAIT_TIME = 10;
   public static final int MINIMUM_SERVER_START_GAME_SYNC_WAIT_TIME =
       MINIMUM_SERVER_OBSERVER_JOIN_WAIT_TIME + ADDITIONAL_SERVER_ERROR_DISCONNECTION_WAIT_TIME + 110;
-  public static final int DEFAULT_SERVER_START_GAME_SYNC_WAIT_TIME =
+  private static final int DEFAULT_SERVER_START_GAME_SYNC_WAIT_TIME =
       Math.max(Math.max(MINIMUM_SERVER_START_GAME_SYNC_WAIT_TIME, 900),
           DEFAULT_SERVER_OBSERVER_JOIN_WAIT_TIME + ADDITIONAL_SERVER_ERROR_DISCONNECTION_WAIT_TIME + 110);
 
   private static final String MAP_FOLDER = "mapFolder";
 
+  private static String[] COMMAND_LINE_ARGS =
+      {TRIPLEA_GAME_PROPERTY, TRIPLEA_SERVER_PROPERTY, TRIPLEA_CLIENT_PROPERTY, TRIPLEA_HOST_PROPERTY,
+          TRIPLEA_PORT_PROPERTY, TRIPLEA_NAME_PROPERTY, TRIPLEA_SERVER_PASSWORD_PROPERTY, TRIPLEA_STARTED,
+          LobbyServer.TRIPLEA_LOBBY_PORT_PROPERTY,
+          LOBBY_HOST, LOBBY_GAME_COMMENTS, LOBBY_GAME_HOSTED_BY, TRIPLEA_ENGINE_VERSION_BIN, HttpProxy.PROXY_HOST,
+          HttpProxy.PROXY_PORT, TRIPLEA_DO_NOT_CHECK_FOR_UPDATES, Memory.TRIPLEA_MEMORY_SET, MAP_FOLDER};
+
+
+  private static void usage(GameMode gameMode) {
+    if(gameMode == GameMode.HEADLESS_BOT) {
+      System.out.println("\nUsage and Valid Arguments:\n"
+          + "   " + TRIPLEA_GAME_PROPERTY + "=<FILE_NAME>\n"
```

Comment:
usage is now formatted : )
 not consolidated, but it is easier to manage them both when they are in one spot.